x-pack/plugins/lists/server/services/exception_lists/get_exception_list_item.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  SavedObjectsClientContract,
+  SavedObjectsErrorHelpers,
+} from '../../../../../../src/core/server/';
+import {
+  ExceptionListItemSchema,
+  ExceptionListSoSchema,
+  IdOrUndefined,
+  ItemIdOrUndefined,
+} from '../../../common/schemas';
+
+import { getSavedObjectType, transformSavedObjectToExceptionListItem } from './utils';
+import { NamespaceType } from './types';
+
+interface GetExceptionListItemOptions {
```

Comment:
This same interface appears [here](https://github.com/elastic/kibana/pull/66443/files#diff-afff0a2bb3df0cd8f9fdfc105598cdaeR49), with the exception of `savedObjectsClient` ... was that intentional?

Reply:
Yeah, one is used within a method and the other is a regular function that is more pure in which the client API calls into with a bit of saved state.